File: android-app/src/com/tahanot/entities/StopSign.java
package com.tahanot.entities;

import java.util.ArrayList;

import org.simpleframework.xml.*;

import android.os.*;

import java.io.Serializable;

@Root
public class StopSign {

	@Element
	public GtfsStop RawStop;

	@ElementList
	public ArrayList<StopSignRoute> Routes;
}

File: azure-website/GtfsService/Entities/StopSign.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using GtfsBackend.Entities.Raw;

namespace GtfsBackend.Entities
{
    public class StopSign
    {
        public GtfsStop RawStop;
        public List<StopSignRoute> Routes { get; set; }

        public StopSign()
        {

        }

        public StopSign(GtfsStop rawStop)
        {
            this.RawStop = rawStop;
        }
        
    }
}
